docs/tutorial/getting-started/life-cycle/index.md

```
@@ -147,7 +149,7 @@ import { Elysia } from 'elysia'
 
 new Elysia()
 	.onBeforeHandle(({ query: { name }, status }) => {
-		if(!name) return status(401)
+		if(!name) return status("Unauthorized")
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)